init.lua

```diff
@@ -8,6 +8,9 @@ skins = {}
 skins.modpath = minetest.get_modpath(minetest.get_current_modname())
 skins.default = "character"
 
+skins.use_voxelibre_compat = minetest.global_exists("mcl_skins")
+assert(minetest.global_exists("player_api") or minetest.global_exists("mcl_skins"), "One of player_api or mcl_skins is required.")
```

Comment:
Applying general Minetest code style rules here.
```suggestion
assert(minetest.global_exists("player_api") or minetest.global_exists("mcl_skins"),
	"One of player_api or mcl_skins is required.")
```